clientv3/integration/dial_test.go

```diff
@@ -66,6 +70,89 @@ func TestDialTLSExpired(t *testing.T) {
 	}
 }
 
+// TestDialTLSExpiredReload ensures server reloads expired certs,
+// rejecting client requests, and vice versa.
+func TestDialTLSExpiredReload(t *testing.T) {
```

Comment:
integration/ instead of clientv3/integration since it's testing server behavior that's client independent?